packages/ui/Button/Button.js

```diff
@@ -81,17 +82,22 @@ const ButtonBase = kind({
 		 * with the caveat that if you supply a custom icon, you are responsible for sizing and
 		 * locale positioning of the custom component.
 		 *
-		 * @type {Node}
+		 * Setting this to `true` means the `iconComponent` will be rendered but with empty content.
+		 * This may be useful if you've customized the `iconComponent` to render the icon content
+		 * externally.
+		 *
+		 * @type {Node|Boolean}
 		 * @public
 		 */
-		icon: PropTypes.node,
+		icon: PropTypes.oneOfType([PropTypes.node, PropTypes.bool]),
 
 		/**
 		 * The component used to render the [icon]{@link ui/Button.ButtonBase.icon}.
 		 *
-		 * This component will receive the `size` property set on the Button as well as the `icon`
-		 * class to customize its styling. If [icon]{@link ui/Button.ButtonBase.icon} is not a
-		 * string, this property is not used.
+		 * This component will receive the `size` and `spriteCount` properties and should them.
+		 * This coomponent will also receive the `icon` class to customize its styling.
```

Comment:
97: What should it do with the `size` and `spriteCount` props?

98: `coomponent` -> `component`

Reply:
It should ^handle^ them. How it handles them, is the "coooooomponent's" deal.